org.mpdevil.mpdevil.json

```diff
@@ -0,0 +1,34 @@
+{
+  "app-id": "org.mpdevil.mpdevil",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.38",
+  "sdk": "org.gnome.Sdk",
+  "command": "mpdevil",
+  "finish-args": [
+    "--share=ipc",
```

Comment:
```suggestion
```

AFAIK `fallback-x11` includes this permission.

Reply:
OK, thanks for the tip.

Reply:
> AFAIK `fallback-x11` includes this permission.

@barthalion  Does it? I think I've asked about this before and was told that it doesn't and the docs don't mention this at all.
Taking a quick glance at flatpak/flatpak, I'm unsure if this is the case (though I'm not familiar with flatpak code at all):
- https://github.com/flatpak/flatpak/blob/412c15772f794fd300991a91f2e0fe51f615bf79/common/flatpak-context.c#L967
- https://github.com/flatpak/flatpak/blob/412c15772f794fd300991a91f2e0fe51f615bf79/portal/flatpak-portal.c#L1027
  This one seems to imply the opposite?